AdyenActions/UI/View Controllers/DelegatedAuthentication/DelegatedAuthenticationErrorView.swift

```diff
@@ -52,6 +53,50 @@ internal final class DelegatedAuthenticationErrorView: UIView {
         scopeInstance: self
     )
     
+    internal lazy var troubleshootingTitle: UILabel = .init(
+        style: style.troubleshootingTitleStyle,
+        accessibilityPostfix: "troubleshootingTitle",
+        multiline: false,
+        textAlignment: .center,
+        scopeInstance: self
+    )
+    
+    internal lazy var troubleshootingDescription: UILabel = .init(
+        style: style.troubleshootingDescriptionStyle,
+        accessibilityPostfix: "troubleshootingDescription",
+        multiline: true,
+        textAlignment: .center,
+        scopeInstance: self
+    )
+    
+    internal lazy var troubleshootingButton: SubmitButton = {
+        let button = SubmitButton(style: self.style.troubleshootingButtonStyle)
+        button.addTarget(self, action: #selector(troubleshootingButtonTapped), for: .touchUpInside)
+        button.accessibilityIdentifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "troubleshootingButton")
+        button.preservesSuperviewLayoutMargins = true
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    internal lazy var troubleshootingStackView: UIStackView = {
+        let stackView = UIStackView(
```

Comment:
I think I missed this before but 2 notes on this stackview helper init:
`scopedInstance` is not used.
`cornerRadius` is constant, shouldn't it be the same radius as its parent view?

Reply:
> scopedInstance is not used

Good catch, thanks. Removed it.

Reply:
> cornerRadius is constant, shouldn't it be the same radius as its parent view?

Changed it and setting the corner radius to both the stack view and the background view. 
TBH I'm not sure if it makes a difference as i imagine the bottom most layer to be a square which is clear color(the default for all views) and whatever we place on top of it can have rounded cornered colored stuff. But yeah, no harm in making it set to both.